bookwyrm/templatetags/utilities.py

```diff
@@ -25,7 +25,7 @@ def get_title(book):
         return ""
     title = book.title
     if len(title) < 6 and book.subtitle:
-        title = "{:s}: {:s}".format(title, book.subtitle)
+        title = "{:s} ({:s})".format(title, book.subtitle)
```

Comment:
Using parens for `Title (Subtitle)` scans oddly to me for English-language titles. I think it would be better to treat this as a translation using `gettext`, with "Title: Subtitle" as the default translation?

Reply:
I agree with the gettext but I won’t take the time to look into it at this point. Do you prefer I revert my changes or keep it like this?

Reply:
Scratch my previous comment. ’Not being in a good place at the moment and I needed a productive distraction.

I added the string and the filter can now take a variable from the template, like so:
This could actually be a variable that can be set for each language, as limits are likely to differ from one to another.


```django
<a href="{{ status.book.local_path }}">{{ status.book|book_title:10 }}</a>
```

Reply:
I like it!